FakeServer/Common/QueryHelper.cs

```diff
@@ -47,6 +47,24 @@ public static class QueryHelper
     {
         public static PaginationHeader GetPaginationHeader(string url, int totalCount, int skip, int take, string skipWord, string takeWord)
         {
+            if (skipWord == "page" && takeWord == "per_page")
+            {
+                int totalPages = totalCount / take;
```

Comment:
Rest of the code base uses var instead of specific type so it could be used here.

totalPages calculation could also be a single liner, like
```csharp
var totalPages = totalCount / take + (totalCount % take != 0 ? 1 : 0);
var page = (skip / take) + 1;

return new PaginationHeader
```

Then the code is faster to read, as you can just see that totalpages is calculated in this line. Naturally then if you care how that is calculated it is more difficult to understand from a single line of code, but often you just wan't to read codebase fast.

Reply:
Cool, that does look nice in one line. I made these changes as you suggested.